### DISCUSSION TITLE
AttributeError: root on self.__pydantic_serializer__.to_python after upgrade Fastapi to 0.103.1

### DISCUSSION THREAD

**Discussion by liguifan** [fastapi/fastapi#10696] [Questions]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

After update fastapi to 0.103.1, the RootModel cannot be serialized anymore. Here is my SerializableMessage class:

```
class SerializableMessage(RootModel):
    root: Message

    class Config:
        arbitrary_types_allowed = True  # for Message
```

the below `server_debug_summary` contains SerializableMessage:

> File "N:\Pycharm\xxxx\src\app\web\views\debug_view.py", line 54, in _compute_debug_summary
>     debug_view_data.update(jsonable_encoder(server_debug_summary))
>   File "N:\Pycharm\xxxx\venv\lib\site-packages\fastapi\encoders.py", line 131, in jsonable_encoder
>     obj_dict = _model_dump(
>   File "N:\Pycharm\xxxx\venv\lib\site-packages\fastapi\_compat.py", line 171, in _model_dump
>     return model.model_dump(mode=mode, **kwargs)
>   File "N:\Pycharm\xxxx\venv\lib\site-packages\pydantic\main.py", line 308, in model_dump
>     return self.__pydantic_serializer__.to_python(
> AttributeError: root

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean and organized.
If this is still relevant to current version, please provide [MRE](https://stackoverflow.com/help/minimal-reproducible-example) and re-open this discussion
